### PR TITLE
[Improvement]add lint

### PR DESCRIPTION
**Pull request by wreulicke**
This PR aims to add tool for linting.
[ESLint](http://eslint.org/) is lint tool for JavaScript.

# add Lint.
Following files is added due to lint :
* ".eslintignore" is a file for Ignoring files from linting.
* ".eslintrc" is setting for linting.

# module dependency.
Following files is added due to collect dependencies for linting :
* In package.json , there are dependencies for npm.

# usage
If you do not have npm and node, please install npm and node.

## lint
you can use a following command for linting.

```bash
npm run lint
```

## format

F○cking codes will make you bore.
ESlint provides formatting feature.  :D
you can use a following command for formatting.

```bash
npm run format
```

-----------------------

Any questions? :D

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1 @@
+pegmatite/rawdeflate.js
```
Comment (dai0304) [inline .eslintignore]: New line at EOF

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,4 @@
 pegmatite.crx
 pegmatite.pem
 pegmatite.zip
+node_modules
```
Comment (dai0304) [inline .gitignore]: NL

File: package.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+	"author": "dai0304",
+	"license": "Apache-2.0",
+	"bugs": {
+		"url": "https://github.com/dai0304/pegmatite/issues"
+	},
+	"homepage": "https://github.com/dai0304/pegmatite#readme",
+	"devDependencies": {
+		"eslint": "^3.19.0"
+	}
+}
```
Comment (dai0304) [inline package.json]: NL

**Comment by wreulicke** [comment]
I added ci settings.

**Comment by wreulicke** [comment]
Sorry, I wroted wrong commands in CI settings.
Fixed.

:D

**Pull request merged**